test/functional/feature_logging.py

```diff
@@ -99,6 +99,18 @@ def run_test(self):
             match=ErrorMatch.PARTIAL_REGEX,
         )
 
+        self.log.info("Test that -nodebug,-debug=0,-debug=none clear previously specified debug options")
+        disable_debug_options = [
+            '-debug=0',
+            '-debug=none',
+            '-nodebug'
+        ]
+
+        for debug_opt in disable_debug_options:
+            self.restart_node(0, ['-debug=http', debug_opt, '-debug=rpc'])
+            logging = self.nodes[0].logging()
+            assert logging['rpc']
```

Comment:
nit: Could add invalid `abc` category prior to disabling, and verify that *multiple* categories after the disabling work.
```suggestion
            self.restart_node(0, ['-debug=http', '-debug=abc', debug_opt, '-debug=rpc', '-debug=net'])
            logging = self.nodes[0].logging()
            assert logging['rpc']
            assert logging['net']
```

Reply:
Nice touch on adding a check for non-existence of abc!
https://github.com/bitcoin/bitcoin/blob/bb8309723cdd3ce0378fbe9708bd600dfaa157aa/test/functional/feature_logging.py#L113